portal/components/react-admin/customHasura/customFields.js

```diff
@@ -21,6 +21,21 @@ const EXTENDED_DONATION_RECORD = gql`
     }
   }
 `;
+const EXTENDED_CORPORATE_DEVICES_RECORD = gql`
```

Comment:
Can we refactor this to include every custom graphQL fragment in a separate file in a `fragments` directory inside `customHasura`?